app/src/main/res/values/strings.xml

```diff
@@ -198,24 +198,24 @@
     <!-- This is used in the app logs to inform that network connectivity has successfully been detected -->
     <string name="network_connection_success">Network connection successful.</string>
 
-    <!-- This is used to inform the user that there are no entries in the list -->
-    <string name="no_entries_found">No Entries Found</string>
-    <!-- This explains the steps the user should take, if no saved entries are found: sign in and sync in Firefox. %1$s will be replaced with application name -->
-    <string name="no_entries_description">%1$s lets you access passwords you’ve already saved to Firefox. To
-        view your entries here, you’ll need to sign in and sync in Firefox.
+    <!-- This is used to inform the user that there are no saved logins in the list -->
+    <string name="no_logins_found">No Logins Found</string>
+    <!-- This explains the steps the user should take, if no saved logins are found: sign in and sync in Firefox. %1$s will be replaced with application name -->
+    <string name="no_logins_description">%1$s lets you access passwords you’ve already saved to Firefox. To
+        view your logins here, you’ll need to sign in and sync in Firefox.
     </string>
     <!-- This is the title for the biometric unlock onboarding screen. %1$s will be replaced with application name -->
     <string name="onboarding_unlock_title">Unlock %1$s with your fingerprint.</string>
     <!-- This is the onboarding description for enabling biometric unlock with the fingerprint sensor. %1$s will be replaced with application name -->
-    <string name="onboarding_unlock_description">Give %1$s permission to unlock this app with the fingerprint sensor.</string>
+    <string name="onboarding_unlock_description">Give %1$s permission to unlock this\napp with the fingerprint sensor.</string>
```

Comment:
Is there a way to not require a manual line break here?

If not, this is another case where being able to verify with actual screenshot tests goes a long way.

Reply:
Good call out here, thanks.

After discussing with @sashei and @eliserichards we will not use a newline and instead explore  some layout constraints to get closer to the intended look (and line wrapping).